examples/summarization/train.py

```diff
@@ -180,8 +193,12 @@ def add_newline_to_end_of_each_sentence(x: str) -> str:
         else:
             return aggregator._scores  # here we return defaultdict(list)
 
+    @rank_zero_only
     def on_end_val_epoch(self, global_step, *collated_output, key="default"):
-        print('Evaluating gathered results.')
+        logger.info('Evaluating gathered results.')
+        if len(collated_output) == 0:
```

Comment:
Why is this required. Unnecessary error check ? We will always have more than zero validation examples collected

Reply:
this is something I think that was more required when Han was running and wasn't using @rank_zero_only, now that we are doing that I can get rid of this.